libs/store/src/lib/infrastructure/persistence/query/query.ts

```diff
@@ -19,6 +20,8 @@ export interface OrderBy <TModel> {
  */
 export class Query<TModel> {
 
+    result$: Subject<Array<TModel>>;
```

Comment:
We shouldn't expose a subject to a public API directly, let's use `asObservable()`